phpBB/styles/prosilver/theme/colours.css

```diff
@@ -640,30 +640,18 @@ Colours and backgrounds for buttons.css
 .contact-icons a		{ border-color: #DCDCDC; }
 .contact-icons a:hover	{ background-color: #F2F6F9; }
 
-/* Pagination
----------------------------------------- */
-
-.pagination li a {
+.button-paging {
 	background: #ECEDEE;
 	filter: none;
 	border-color: #B4BAC0;
 	box-shadow: none;
 	-webkit-box-shadow: none;
-	color: #5C758C;
-}
-
-.pagination li.ellipsis span {
-	background: transparent;
-	color:	#000000;
-}
-
-.pagination li.active span {
-	background: #4692BF;
-	border-color: #4692BF;
-	color: #FFFFFF;
+	color: #8f8f8f;
 }
 
-.pagination li a:hover, .pagination li a:hover .icon, .pagination .dropdown-visible a.dropdown-trigger, .nojs .pagination .dropdown-container:hover a.dropdown-trigger {
+.button-paging:hover,
+.button-paging:focus,
+.button-active {
```

Comment:
Missing the last selectors whose purpose is to keep the page jump button active while the dropdown is open.